TEST-198 Update smarter testing onboarding docs (#10511)
circleci/circleci-docs: docs/guides/modules/test/pages/getting-started-with-smarter-testing.adoc

== Before you begin

* You need to install the latest xref:toolkit:local-cli.adoc[Local CLI].

> * You need a repository with an existing test suite. The setup process guides you through configuring JUnit XML output and installing the CLI `testsuite` plugin.

Reviewer: Is that the "setup process" described on this page? If so can we add that to this line

Reviewer 2: Would "The following setup process..." work?

--
====

== 2. Validate and run locally

> Use the `--doctor` flag to set up and validate your `test-suites.yml`. The CLI runs through a series of checks, executing tests and validating the output. If any results look incorrect, an action item is provided to resolve it.

Reviewer: ```suggestion
Use the `--doctor` flag to set up and validate your `test-suites.yml`. The CLI runs through a series of checks, executing tests and validating the output. If any results look incorrect, action items are provided to resolve them.
```

include::ROOT:partial$tips/testsuite-working-directory.adoc[]

== 3. Run in CI

include::ROOT:partial$tips/testsuite-ci-preinstalled.adoc[]

> In your `.circleci/config.yml`, replace your existing test command with `circleci run testsuite`. Keep all other job steps the same.

Reviewer: ```suggestion
In your CIrcleCI configuration file, replace your existing test command with `circleci run testsuite`. Keep all other job steps the same.
```
